CSS properties: intro sentence in values section part 3 (#44936)
mdn/content: files/en-us/web/css/reference/properties/caret-color/index.md

### Values

> This property is specified as one `<color>` value or the keyword `auto`:

Reviewer: ```suggestion
This property is specified as a single value from the following list:
```
